src/components/Display.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropType from 'prop-types';
+
+const Display = ({ result = '0' }) => (
+  <div>
```

Comment:
Normally, this should be enforced with `defaultProps`

Reply:
thanks, I forgot it!